src/transformers/modeling_utils.py

```
@@ -2251,10 +2259,15 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                 torch_dtype = torch.float16
 
             if device_map is None:
-                raise ValueError(
```

Should raise an error here if there is no GPU maybe?